### PR TITLE
Current measurement

### PR DESCRIPTION
**Pull request by RhushabhRR**
Different techniques of current measurement added

### CODE REVIEW

File: docs/development/current_measurement.md [hunk 1]
```diff
@@ -1,9 +1,177 @@
...
+    <img src="./images/hall_current_measurement.svg" height="auto" width="auto" />
+    <figcaption><b>Figure 4.</b> Principle of hall sensor based current measurement.</figcaption>
+</center>
+</figure>
+
+### 3. Inductive sensor
+
+The [inductive sensor](https://en.wikipedia.org/wiki/Inductive_sensor) consists of a wire-wound core through which current carrying conductor passess. AC current flowing through this conductor constantly changes potential from positive to negative and back again. This results in expanding and collapsing magnetic field which induces voltage in the windings. Further inductive sensor converts this induced voltage into proportional voltage which will be given as output. Hence, in this case, voltage is the measure of change in magnetic field strength and not direct measure of magnetic field as in the case of Hall effect sensor. That's why inductive sensors are only suitable for measuring AC currents over other methods. Like Hall sensor, this also provides excellent isolation from the main circuit and has the advantage of very low noise when compared with the former. 
+
+#### Conclusion
```
Comment (martinjaeger) [inline docs/development/current_measurement.md]: Should be ### as this should be on the same indentation level as the previous three sections.

File: docs/development/current_measurement.md [hunk 2]
```diff
@@ -1,9 +1,177 @@
...
+
+<figure>
+<center>
+    <img src="./images/high_side_current_measurement.svg" height="auto" width="auto" />
+    <figcaption><b>Figure 7.</b> Principle of high-side current measurement.</figcaption>
+</center>
+</figure>
+
+When higher supply voltages (30-40V) are used, the circuit needs to be modified such that low voltage can be measured across the output, so that it can be integrated with other intermediate circuits or microcontroller. The modification depends mainly on the required output volatge range. An example of such circuit is shown below in the Fig.8 where the output voltage has to be within 5V.
```
Comment (martinjaeger) [inline docs/development/current_measurement.md]: - Replace (30-40V) with (>15V)
- Typo in "voltage" at end of second sentence.

File: docs/development/current_measurement.md [hunk 3]
```diff
@@ -1,9 +1,177 @@
...
+
+<figure>
+<center>
+    <img src="./images/modified_high_side_measurement.svg" height="auto" width="auto" />
+    <figcaption><b>Figure 8.</b> Modified high-side current measurement for high voltages.</figcaption>
+</center>
+</figure>
+
+Note - $V_{dd}$ = $V_{CC}$ + 10
```
Comment (martinjaeger) [inline docs/development/current_measurement.md]: Use whole sentence: Note that $V_{dd}$ has to be a few volts higher than $V_{cc}$.

**Review by martinjaeger — CHANGES_REQUESTED** [review]
Almost ready to merge, see some small comments below.

And there is a hall_sensor.gif image left which is not used in the text. Please remove it in the commit that introduced it.

Thanks!

**Comment by RhushabhRR** [comment]
> Almost ready to merge, see some small comments below.
> 
> And there is a hall_sensor.gif image left which is not used in the text. Please remove it in the commit that introduced it.
> 
> Thanks!

Hi Martin, for removing the hall_sensor.gif in the commit it was introduced, I get a detached head when i pull an old commit. The problem is I can't push the changes from an old commit. Is there any way to do it?
Other changes have been made

**Comment by martinjaeger** [comment]
Don't worry. Updated it here and will merge now.

**Review by martinjaeger — APPROVED** [review]

**Pull request merged**